src/test/java/cydeo/emrah/carBrand.txt

```diff
@@ -0,0 +1,3 @@
+Car brand
+Ford
+Audi
```

Comment:
@bltemrah 
I think you forgot Mercedes as I remember one of thr AC says Mercedes should accept as one of the car brand

Reply:
Hey @Aitats33 
Yes, I will add it. thanks